### PR TITLE
tickets/DM-44757

### PR DESCRIPTION
**Pull request by elanaku**
Edited fiber spectrograph code to take a group ID.

### CODE REVIEW

File: python/lsst/ts/fiberspectrograph/csc.py [hunk 1]
```diff
@@ -249,9 +249,9 @@ async def do_expose(self, data):
             await self.fault(code=20, report=msg)
             raise salobj.ExpectedError(msg)
 
-        await self.save_data(spec_data)
+        await self.save_data(spec_data, data.groupId)
```
Comment (tribeiro) [inline python/lsst/ts/fiberspectrograph/csc.py]: As I mentioned in the electrometer PR, you might want to ensure this change is backwards compatible, for the time being.

**Review by tribeiro — COMMENTED** [review]
As mentioned in the electrometer PR, make sure this change is backwards compatible. Also, can you add the news fragments for this change?

**Review by parfa30 — APPROVED** [review]
Looks good to me

**Review by parfa30 — APPROVED** [review]
Please squash commits. Looks good!

**Pull request merged**